pom.xml

```diff
@@ -1061,15 +1378,21 @@
                                     <include>org.wildfly.security.key</include>
                                     <include>org.wildfly.security.manager</include>
                                     <include>org.wildfly.security.manager.action</include>
+                                    <include>org.wildfly.security.manager._private</include>
                                     <include>org.wildfly.security.mechanism</include>
+                                    <include>org.wildfly.security.mechanism.gssapi</include>
+                                    <include>org.wildfly.security.mechanism._private</include>
                                     <include>org.wildfly.security.password</include>
                                     <include>org.wildfly.security.password.interfaces</include>
                                     <include>org.wildfly.security.password.spec</include>
                                     <include>org.wildfly.security.permission</include>
+                                    <include>org.wildfly.security.sasl.auth.util</include>
                                     <include>org.wildfly.security.sasl.util</include>
                                     <include>org.wildfly.security.ssl</include>
+                                    <include>org.wildfly.security.ssl._private</include>
                                     <include>org.wildfly.security.x500</include>
                                     <include>org.wildfly.security.x500.cert</include>
+                                    <include>org.wildfly.security.x500.principal</include>
```

Comment:
I added modules noted Public from here: https://docs.google.com/spreadsheets/d/11B6rRnW18gL5zTBuyC-qxb8rMfpF7GfVWxrWFZqRiaQ/edit#gid=0 . Those that are marked Mixed I was not sure about which should  be added here.

Reply:
For cross referencing purposes you may be better looking at the included set of packages here https://github.com/wildfly/wildfly-core/blob/master/core-feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/security/elytron/main/module.xml

These are the ones considered public API.

Reply:
I thought these were not updated? Judging by Farah's comment here: https://github.com/wildfly-security-incubator/wildfly-elytron/pull/63#issuecomment-452039722

Reply:
@Skyllarr It's safe to look at the list Darran mentioned but that list doesn't contain any new public packages that have been introduced in the module split (e.g., ```org.wildfly.security.auth.server.http```, ```org.wildfly.security.auth.server.sasl```).

Reply:
For this first step I think it is safe to ignore the new packages, it is when we release 1.9.0.Final that we will need to start checking those as well.

Reply:
Ah, yes, my linked comment was about the ```sourceFileIncludes``` for the ```api-javadocs```. For JAPICMP, it makes sense to ignore the new ones for now.

Reply:
Thank you. So the list did not change for this one. I updated the PR.

Reply:
Just an FIY, since we did ignore new packages for now, output returns that eg. class org.wildfly.security.sasl.util.AuthenticationContextSaslClient became deprecated,  but class org.wildfly.security.sasl.**auth**.util.AuthenticationContextSaslClient is not listed among added classes (package sasl.auth is ignored).